exercises/dominoes/tests/dominoes.rs

```diff
@@ -6,9 +6,9 @@ type Domino = (usize, usize);
 
 #[derive(Debug)]
 enum CheckResult {
-    GotInvalid,             // chain returned None
+    GotInvalid, // chain returned None
     Correct,
-    ChainingFailure(Vec<Domino>), // failure to match the dots at the right side of one domino with
+    ChainingFailure(Vec<Domino>), /* failure to match the dots at the right side of one domino with */
```

Comment:
To make clear, I would not hold up the PR over this specific change, but am curious to know why it happened